### PR TITLE
Add Sourcery action

### PR DESCRIPTION
**Pull request by BohdanBilonoh**
Installation of Sourcery if it's missed.

### CODE REVIEW

File: sourcery/sourcery.rb [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      def self.details
+        "
+        - Install Sourcery via Homebrew
+        - Run Sourcery
+        "
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(
```
Comment (AndriiMoskvin) [inline sourcery/sourcery.rb:39]: All params have to be specified here.

File: sourcery/sourcery.rb [hunk 2]
```diff
@@ -0,0 +1,62 @@
+module Fastlane
+  module Actions
+    class SourceryAction < Action
+      def self.run(params)
+        action = params[:action]
+
+        case action
+        when "install"
+            require 'brew'
+            brew(command: 'install sourcery')
```
Comment (AndriiMoskvin) [inline sourcery/sourcery.rb:10]: 👍

File: sourcery/sourcery.rb [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        when "run"
+            sources = params[:sources]
```
Comment (AndriiMoskvin) [inline sourcery/sourcery.rb:12]: All this params have to be specified in `FastlaneCore::ConfigItem`.

**Review by AndriiMoskvin — CHANGES_REQUESTED** [review]

File: sourcery/sourcery.rb [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+              when "install"
+                true
+              when "run"
+                true
+              else
+                UI.user_error!("Don't support action: #{value}")
+              end
+            end
+          ),
+          FastlaneCore::ConfigItem.new(
```
Comment (AndriiMoskvin) [inline sourcery/sourcery.rb:55]: All this items are required for "install"?

**Review by AndriiMoskvin — CHANGES_REQUESTED** [review]

File: sourcery/sourcery.rb (hunk 1 above)
Comment (BohdanBilonoh) [reply sourcery/sourcery.rb:39]: @AndriiMoskvin Done

File: sourcery/sourcery.rb (hunk 3 above)
Comment (BohdanBilonoh) [reply sourcery/sourcery.rb:12]: @AndriiMoskvin Done

File: sourcery/sourcery.rb (hunk 4 above)
Comment (BohdanBilonoh) [reply sourcery/sourcery.rb:55]: @AndriiMoskvin Set `optional: true`

**Review by AndriiMoskvin — APPROVED** [review]

**Pull request merged**